offchain-modules/packages/x/src/handlers/ada.ts

```diff
@@ -173,7 +174,8 @@ export class AdaHandler {
                 await this.handleTx(tx, currentHeight);
                 if (confirmed) {
                   lastConfirmedTxTime = tx.inserted_at.time;
-                  lastConfirmedTxHash = tx.id;
+                  /// caused linter error "never used"
+                  // lastConfirmedTxHash = tx.id;
```

Comment:
If it isn't used I think it is safe to just remove all of this

Reply:
Ok, I will do that